acs-integration-tests/src/test/java/com/ge/predix/acceptance/test/ACSAcceptanceIT.java

```diff
@@ -79,74 +71,44 @@
     private PolicyHelper policyHelper;
 
     @Autowired
-    private ZacTestUtil zacTestUtil;
-
-    @Autowired
-    private Environment env;
-
-    @Autowired
     private ACSRestTemplateFactory acsRestTemplateFactory;
 
-    @Value("${ZONE1_NAME:testzone1}")
-    private String acsZone1Name;
-
-    @Value("${ZONE2_NAME:testzone2}")
-    private String acsZone2Name;
-
-    @Value("${ZONE3_NAME:testzone3}")
-    private String acsZone3Name;
-
-    @Value("${ACS_UAA_URL}")
+    @Value("${ZONE1_UAA_URL}")
     private String uaaUrl;
 
     private OAuth2RestTemplate acsAdminRestTemplate;
-    private boolean registerWithZac;
+    private OAuth2RestTemplate acsZoneRestTemplate;
+    private HttpHeaders headersWithZoneSubdomain = new HttpHeaders();
+    private UaaTestUtil uaaTestUtil;
+    private String acsZoneName;
 
     @BeforeClass
-    public void setup() throws JsonParseException, JsonMappingException, IOException {
-        if (Arrays.asList(this.env.getActiveProfiles()).contains("public")) {
-            setupPublicACS();
-        } else {
-            setupPredixACS();
-        }
-        this.headersWithZoneSubdomain = new HttpHeaders();
-        this.headersWithZoneSubdomain.set("Predix-Zone-Id", this.testZoneSubdomain);
-    }
-
-    private void setupPredixACS() throws JsonParseException, JsonMappingException, IOException {
-        this.zacTestUtil.assumeZacServerAvailable();
-
-        this.acsAdminRestTemplate = this.acsRestTemplateFactory.getACSTemplateWithPolicyScope();
-        this.registerWithZac = true;
-        this.testZoneSubdomain = this.zoneHelper
-                .createTestZone(this.acsAdminRestTemplate, this.acsZone1Name, this.registerWithZac).getSubdomain();
-    }
+    public void setup() throws IOException {
+        this.acsZoneName = this.zoneHelper.getRandomName(this.getClass().getSimpleName());
+        this.headersWithZoneSubdomain.set("Predix-Zone-Id", this.acsZoneName);
```

Comment:
I take it back... :(
Turns out Predix-Zone-Id is not the only header name that could be used to set the zone name.  ACS is setup to also use ACS-Zone-Subdomain.  By using @value, we could test public profile with ACS-Zone-Subdomain while testing predix profile with Predix-Zone-Id.